src/Microsoft.Azure.WebJobs.Host/Extensions/IWebjobsExceptionHandlerExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.ExceptionServices;
+
+namespace Microsoft.Azure.WebJobs.Host.Timers
+{
+    /// <summary>
+    /// Helper methods for IWebJobsExceptionHandler
+    /// </summary>
+    public static class IWebJobsExceptionHandlerExtensions
```

Comment:
I think the convention is to actually *keep* the I since this is an extension class. 

Consider if you had class Foo, and interface IFoo, and then extensions classes on each, you'd need "FooExtension" and "IFooExtension" to avoid conflict.